Realm/Realm/DatabaseTypes/Metadata.cs

```diff
@@ -42,5 +42,15 @@ public Metadata(TableKey tableKey, IRealmObjectHelper helper, IDictionary<string
             PropertyIndices = new ReadOnlyDictionary<string, IntPtr>(propertyIndices);
             Schema = schema;
         }
+
+        public static IntPtr GetPropertyIndex(string propertyName, Metadata metadata)
```

Comment:
This should be an instance method, since it's on the `Metadata` class itself, instead of a static method that takes a `Metadata` instance argument.